wordpress.org/public_html/wp-content/plugins/photo-directory/inc/uploads.php

```
@@ -703,16 +703,23 @@ protected static function validate_upload_form() {
+
+		foreach ( $fields as $field => $sanitize ) {
+			// phpcs:ignore WordPress.Security.NonceVerification.Missing, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- Sanitized on the next line, by whichever callback stores the field.
 			$submitted = isset( $_POST[ $field ] ) ? wp_unslash( $_POST[ $field ] ) : '';
 
 			// A field can arrive as an array, which Frontend Uploader drops before it builds the post.
 			if ( ! is_string( $submitted ) || '' === $submitted ) {
 				continue;
 			}
 
-			if ( preg_match( '/' . get_shortcode_regex() . '/', $submitted ) ) {
+			if ( 1 === preg_match( '/' . get_shortcode_regex() . '/', $sanitize( $submitted ) ) ) {
```

PHPStan (2.2.8)

[error] Trying to invoke 'sanitize_text_field'|'sanitize_textarea…' but it's not a callable.


(callable.nonCallable)